todo/middlewares/jwt_auth.py

```
@@ -73,25 +82,61 @@ def _try_google_auth(self, request) -> bool:
+                try:
+                    payload = validate_google_access_token(google_token)
+                    self._set_google_user_data(request, payload)
+                    return True
+                except (GoogleTokenExpiredError, GoogleTokenInvalidError):
+                    pass
+
+            return self._try_google_refresh(request)
+
+        except (GoogleTokenExpiredError, GoogleTokenInvalidError) as e:
+            raise e
+        except Exception:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)